extensions/dicom-seg/src/components/BrushRadius.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Range } from '@ohif/ui';
+
+const BrushRadius = ({ value, onChange }) => (
+  <div className="brush-radius">
+    <label htmlFor="brush-radius">Brush Radius</label>
+    <Range
+      value={value}
+      min={1}
```

Comment:
cornerstoneTools has a min and max brush radius, so we should read it from there as a consumer may set it up differently.

```js
const {configuration} = cornerstoneTools.getModule('segmentation');
const {minRadius, maxRadius} = configuration;
```

@igoroctaviano  @dannyrb, This doesn't have to go in now, but perhaps we should should wrap get/set interactions with the cornerstoneTools store with context to make these kind of things less blurry?

We don't have to worry about this right now, as we will add the brush radius slider back in when we add creation functionality.

Reply:
I have added a todo for it and also added the minradius, maxradius from config. I think it's a good to have these interactions indirectly, allows us to decouple this component from cstools.